resources/schema.gql

```diff
@@ -1163,6 +1164,14 @@ type DatasetData {
 	"""
 	estimatedSize: Int!
```

Comment:
My naive expectation is to have 3 sizes:
 - direct data size
 - linked data size
 - total data size = direct + linked

I'm not sure on the motivation for separate size counting, but I think UI should highlight the total size, since that would be the future metric for storage billing.

It's probably the total size interested whe estimating how much disk space will be consumed when thinking "what if I pull this dataset locally".